File: Python/bronze/b1/평균은 넘겠지.py
n = int(input())

for i in range(n):
    num = list(map(int, input().split()))

    avg = sum(num[1:])/num[0]
    count = 0

    for j in range(num[0]):
        if avg < num[j+1]:
            count = count + 1

    print('{0:.3f}%'.format( round( (count / num[0]) * 100,3) ) )

File: kotlin_java/src/baekjoon/bronze/b1/평균은 넘겠지.kt
package baekjoon.bronze.b1

import java.lang.StringBuilder
import kotlin.math.roundToInt

fun main() = with(System.`in`.bufferedReader()) {
    val n = readLine().toInt()
    val sb = StringBuilder()
    repeat(n) {
        val arr = readLine().split(" ").map { it.toInt() }
        val avg = (arr.sum() - arr[0]) / arr[0]
        var count = 0
        for (i in 1..arr[0]) {
            if (arr[i] > avg) count++
        }

        sb.append("${String.format("%.3f", (count.toDouble() / arr[0] * 100000.0).roundToInt() / 1000.0)}%\n")
    }
    print(sb)
}